.github/workflows/dart_ci.yml

```
@@ -38,8 +38,12 @@ jobs:
         if: always() && ${{ matrix.sdk }} == '2.7.2' && steps.install.outcome == 'success'
 
       # Analyze before generated files are created to verify that component boilerplate analysis is "clean" without the need for building
-      - name: Analyze project source (pre-build)
-        run: pub run dart_dev analyze
+      - name: Analyze example source (pre-build)
+        run: |
+          cd example/boilerplate_versions
```

I just realized one thing; we used to perform analysis on lib pre-build, but now we don't. 

This step helps ensure that the package's public APIs don't depend on build-to-cache files, which would break consumers who haven't run builds yet.

Could we add that back in?
```suggestion
          # Analyze lib to ensure public APIs don't depend on build-to-cache files, 
          # which could cause analysis issues for consumers who haven't run a build yet.
          dartanalyzer lib
          cd example/boilerplate_versions
```